hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -249,6 +249,21 @@
           + "record size estimate compute dynamically based on commit metadata. "
           + " This is critical in computing the insert parallelism and bin-packing inserts into small files.");
 
+  public static final ConfigProperty<String> MAX_ARCHIVE_FILES_TO_KEEP_PROP = ConfigProperty
+      .key("hoodie.max.archive.files")
```

Comment:
nit: after thinking about the naming again, let's use `hoodie.archive` prefix for the archive configs and update the variable naming accordingly.
For this one, it can be `hoodie.archive.max.files`.

Reply:
+1

Reply:
This falls under archival timeline so better to have the same `hoodie.archive` prefix.  Given that the writer archives instants instead of files, this shouldn't create confusion.

Reply:
Changed.